src/components/Search/SearchResultsListItem.tsx

```diff
@@ -82,14 +87,26 @@ function SearchResultsListItem({
           disablePadding
         >
           <ListItemIcon className='search-revision-item-icon search-revision'>
-            <Checkbox
-              className='search-revision-item-checkbox'
-              edge='start'
-              tabIndex={-1}
-              disableRipple
-              data-testid={`checkbox-${index}`}
-              checked={isChecked}
-            />
+            {searchType === 'base' ? (
+              <Radio
+                className='search-revision-item-checkbox'
+                edge='start'
+                tabIndex={-1}
+                disableRipple
+                // name='radio-button'
+                data-testid={`radio-${index}`}
+                checked={isChecked}
+              />
+            ) : (
+              <Checkbox
+                className='search-revision-item-checkbox'
+                edge='start'
+                tabIndex={-1}
+                disableRipple
+                data-testid={`checkbox-${index}`}
```

Comment:
ideally we shouldn't use these test ids...
TBH I'm fine if you want to keep "checkbox" for all of them, because that's test only...